modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -157,7 +157,8 @@ To enable external storage, as the ownCloud administrator go to menu:Settings[St
 .When you create a new WND share, you need three things:
```

Comment:
```suggestion
.When you create a new WND share, you need multiple things:
```
We go from three to four here, but "credentials" was already 'two things' - in my world.